front/app/components/Form/Components/Layouts/CLSurveyPageLayout.tsx

```diff
@@ -100,12 +79,6 @@ const CLSurveyPageLayout = memo(
     const { data: phase } = usePhase(phaseId);
     const allowAnonymousPosting =
       phase?.data.attributes.allow_anonymous_participation;
-    const { slug } = useParams();
-    const { data: project } = useProjectBySlug(slug);
-    const { data: authUser } = useAuthUser();
-    const userIsModerator =
-      !isNilOrError(authUser) &&
-      canModerateProject(project?.data.id, { data: authUser.data });
```

Comment:
All of this data loading was here to then just pass it all to SurveyHeading. It's much better to load it in SurveyHeading instead.